src/UI/Areas/Identity/Pages/Account/Manage/TwoFactorAuthentication.cshtml.cs

```diff
@@ -24,7 +24,10 @@ public abstract class TwoFactorAuthenticationModel : PageModel
         [TempData]
         public string StatusMessage { get; set; }
 
-        public virtual Task<IActionResult> OnGet() => throw new NotImplementedException();
+        public virtual Task<IActionResult> OnGetAsync() => throw new NotImplementedException();
```

Comment:
@javiercn can you review the fix for the TFA page that was missing OnPost, is this the correct fix?

Reply:
Yep